conf/application.conf

```diff
@@ -15,7 +15,7 @@
 include "frontend.conf"
 
 appName = "customs-exports-internal-frontend"
-application.router = prod.Routes
+application.router = testOnlyDoNotUseInAppConf.Routes
```

Comment:
Is this effectively the feature switch that prevents these new endpoints being exposed?   Do we need to update one of the other app-confix-xxx projects?

Reply:
Yes, we will allow this on Development, but after merge

Reply:
This should be made only on app-config-development